### ISSUE TITLE
Accessibility: Broken skip link on Item Detail page

### ISSUE THREAD

**Issue by VanessaVenti** [projectblacklight/spotlight#3549]
### Issue Description - WCAG standard
The skip link is broken on the Item Detail page.
WCAG Criteria: 2.4.1 Bypass Blocks

### Method of discovery
WAVE

### Suggested solution

Provide a working skip link to jump to the main content from the top of the page
